CorsixTH/Lua/dialogs/fullscreen/annual_report.lua

```diff
@@ -533,7 +534,7 @@ function UIAnnualReport:draw(canvas, x, y)
       end
       self.stone_font:draw(canvas, award_type, x + 220, y + 330, 200, 0)
       -- Amount won/lost
-      self.stone_font:draw(canvas, "+" .. info.amount, x + 220, y + 355, 200, 0)
+      self.stone_font:draw(canvas, "+" .. string.format("%.0f", info.amount), x + 220, y + 355, 200, 0)
```

Comment:
concatenation is unneeded, there is a format specifier to include sign: https://www.cplusplus.com/reference/cstdio/printf/